meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -17,4 +18,15 @@ SRC_URI:append:am62xx-evm = " \
     file://lmp.cfg \
 "
 
+PATCHTOOL = "git"
+
 PACKAGECONFIG[optee] = "TEE=${STAGING_DIR_HOST}${nonarch_base_libdir}/firmware/tee-pager_v2.bin,,optee-os-fio"
+
+python () {
+    if os.path.exists("${TOPDIR}/conf/keys/custMpk.pem"):
```

Comment:
So here we're hardcoding the key name, we should have it under a variable instead.

Since this is the RoT key, we can probably make it specific to it, like:

K3_ROT_KEYNAME
K3_ROT_KEYDIR

And set it to ?= "" by default, allowing the customer to replace it.

Reply:
um I dont think we should replace the name of the ke:  it means something for TI, is documented in KeyWriter; the way I see it it doesnt bring any benefits being able to replace the name.

Reply:
Would it make any difference in the end? If you would prefer having a hardcoded name, then we should have at least a customized K3_ROT_KEYDIR (as we would have one dir for the public build and another for the factory build).

Reply:
I just don't think changing the name adds any value - remember that this key must match the one in the keywriter software package (which was given the same name). Matching names just keeps it coherent and easier to follow.

you mean:
`"${TOPDIR}/${K3_ROT_KEYDIR}/custMpk.pem"` ?

Reply:
Sure.

In this case it would be the entire dir, as we have for other similar variables (https://github.com/foundriesio/partner-setup/blob/main/partner_api/templates/meta-subscriber-overrides/conf/machine/include/lmp-factory-custom.inc).

Something like:

`K3_ROT_KEYDIR = "${TOPDIR}/conf/factory-keys"`

Then when using it you can just append /custMpk.pem.

Reply:
ah I get your point, doing it now